akka-http-core/src/main/scala/akka/http/model/HttpEntity.scala

```diff
@@ -160,11 +160,11 @@ object HttpEntity {
     override def toStrict(timeout: FiniteDuration)(implicit fm: FlowMaterializer) =
```

Comment:
The solution is simple: use `Any` for the field’s type. Fixing this particular signature was the purpose of the whole exercise AFAIK, so I don’t think it is optional ;-)